docs/sources/community/lids/__DRAFT__loki-mixin-configuration-improvements.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "XXXX: Loki mixin configuration improvements"
+description: "Improve Loki mixin configurations"
+draft: true
```

Comment:
Setting `draft: true` means that this will not be published by Hugo.

https://grafana.com/docs/writers-toolkit/write/front-matter/#draft

Presumably you want this published and available for discussion?

Reply:
Well yes...

That being said,

I used the template: https://raw.githubusercontent.com/grafana/loki/21cfa331312c2c15d833b7a7098450c329fcb92e/docs/sources/community/lids/template.md

And saw no references to https://grafana.com/docs/writers-toolkit/write/front-matter/#draft in either https://github.com/grafana/loki/blob/21cfa331312c2c15d833b7a7098450c329fcb92e/CONTRIBUTING.md or the template itself.

Is there anything else that looks wrong about this PR?

Reply:
Just saw that the LIDs are actually available at https://grafana.com/docs/loki/latest/community/lids/
That could be explained better :)

Reply:
This is a draft LID so "draft: true" made sense to me. Especially since it was hardcoded as is.

I'm a bit lost about how this process it supposed to work. It's not explained.

This draft is ready.
I'm also ready to discuss it or answer any questions about it.

Reply:
I agree that having `draft:true` is confusing.  
I'll publish this and add it to next week's team agenda, so the developers know about it.